R/optionalInput.R

```diff
@@ -443,20 +443,27 @@ optionalSliderInput <- function(inputId, label, min, max, value, label_help = NU
 #' optionalSliderInputValMinMax("a", "b", 1)
 #' optionalSliderInputValMinMax("a", "b", c(3, 1, 5))
 optionalSliderInputValMinMax <- function(inputId, label, value_min_max, label_help = NULL, ...) { # nolint
+  checkmate::assert(
+    checkmate::check_numeric(
+      value_min_max,
+      min.len = 3,
+      max.len = 3,
+      lower = value_min_max[2],
+      upper = value_min_max[3]
+    ),
+    checkmate::check_numeric(
+      value_min_max,
+      min.len = 1,
+      max.len = 1
+    )
+  )
```

Comment:
why min set to 1? Theoritically this should allow negatives. Otherwise it's good.

Reply:
Oh right, I was thinking about the slider as it is being used in `plot_with_settings`, where negative values would be invalid, but this is a general tool, so the `min` should be removed :+1:

Reply:
Hmmm. I am testing this now and it seems that the message seems to be much more cryptic that it was previously:
```
r$> optionalSliderInputValMinMax("a", "b", c(5, 1, 3))
Error: Assertion failed. One of the following must apply:
 * checkmate::check_numeric(value_min_max[c(2, 1, 3)]):
 * Must be sorted
 * checkmate::check_numeric(value_min_max): Must have
 * length 1, but has length 3
```

It is still correct but much more cluttered. I think it would be better to separate class & length checks from value checks